platform/darwin/src/MGLTypes.h

```diff
@@ -107,6 +108,33 @@ NS_INLINE MGLTransition MGLTransitionMake(NSTimeInterval duration, NSTimeInterva
     return transition;
 }
 
+/**
+ A structure containing information about light position.
+ */
+typedef struct MGLPosition {
```

Comment:
Initially I thought this may be used beyond light. But I can move it to MGLLight.h and refactor in the future if happens to be used elsewhere.